linux/net/rina/connection.c

```diff
@@ -59,6 +59,13 @@ struct conn_policies * conn_policies_create(void)
                 return NULL;
         }
 
+        tmp->dtp_ps = policy_create();
```

Comment:
As discussed previously, what's the point in having this tmp->dtp_ps (which are strings, basically)? It's never used and you have the same info in dtp->dtp_ps. Minor point: you use the same names for both struct dtp_ps and struct policy, and I think this is confusing.